src/pages/Home/Home.jsx

```diff
@@ -27,70 +52,42 @@ export const Home = () => {
       </div>
       {/* App intrduction ends */}
 
-      <div className="grid-category">
-        {/* <!-- || Books type starts--> */}
-        <h2 className="section-heading center-text">Types</h2>
-        <div className="book-type">
-          <div className="type type-fiction">
-            <img
-              src="https://n2.sdlcdn.com/imgs/e/h/m/Fiction-0ba2f.jpg"
-              alt="no preview available"
-            />
-            <p>Fiction</p>
-          </div>
-          <div className="type type-nonfiction">
-            <img
-              src="https://n2.sdlcdn.com/imgs/e/h/m/NonFiction-3fef2.jpg"
-              alt="no preview available"
-            />
-            <p>Non-Fiction</p>
-          </div>
-          <div className="type type-philosophy">
-            <img
-              src="https://n4.sdlcdn.com/imgs/e/h/m/philosphy-ed2c5.jpg"
-              alt="no preview available"
-            />
-            <p>Philosophy</p>
-          </div>
-          <div className="type type-family">
-            <img
-              src="https://n3.sdlcdn.com/imgs/e/h/m/family-fcca1.jpg"
-              alt="no preview available"
-            />
-            <p>Family & Relationship</p>
+      {categories.length !== 0 || homeProducts.length !== 0 ? (
+        <div class="grid-category">
+          <h2 className="section-heading center-text">Types</h2>
+          <div class="book-type">
+            {categories.map((item) => {
```

Comment:
for an extra check, we can add `categories &&` before `categories.map` to make sure categories are being fetched from API otherwise sometimes it gives an error of `undefined`